geometry_msgs/Pose target
---
trajectory_msgs/JointTrajectory joint_trajectory
